packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,9 +6,11 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Added optional `patternMatches` property to the `IZoweDatasetTreeNode` interface to cache pattern matches from an applied filter search. [#1164](https://github.com/zowe/zowe-explorer-vscode/issues/1164)
+
 ### Bug fixes
 
-- Fixed errors being logged silently rather than thrown in `ProfilesCache.refresh` method. [#3066](https://github.com/zowe/zowe-explorer-vscode/issues/3066)
+- Fix extender's ability to fetch profile information from ProfilesCache for SSH profile types.
```

Comment:
Curious if we should add this PR as a link.
Also, wondering how the regression occurred.